nni/retiarii/execution/api.py

```diff
@@ -43,6 +44,12 @@ def submit_models(*models: Model) -> None:
     engine.submit_models(*models)
 
 
+def list_models(*models: Model) -> Iterable[Model]:
```

Comment:
I think it's good to make return type concrete, unless it's likely to change in future.

Reply:
I think iterable is fine. Some execution engine might favor to return a generator since there are a lot of models.